src/main/java/step3/model/lotto/Lotto.java

```diff
@@ -0,0 +1,55 @@
+package step3.model.lotto;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import step3.model.util.InputValidation;
+import step3.model.value.Rule;
+
+public class Lotto{
+    private List<LottoNumber> lotto;
+    public Lotto(String input) {
+        InputValidation.validateEmpty(input);
+        List<String> lottoNumbers = splitLottoNumbers(input);
+        this.lotto = createLotto(lottoNumbers);
```

Comment:
final 선언과
```suggestion
        this.lotto = Collections.unmodifiableList(createLotto(lottoNumbers));
```
를 통해 불변 List를 만들 수 있어요.

참고하실만한 링크 보내드릴게요
https://ecsimsw.tistory.com/795

Reply:
불변리스트라는 것은 처음 들어봤는데, 알려주셔서 감사합니다. !!
확실히 기억하고, 다음부턴 변하면 안되는 컬렉션의 경우 꼭 적용하도록 하겠습니다.